### ISSUE TITLE
Command Break\Pause Important!

### ISSUE THREAD

**Issue by CryptoNickSoft** [jcfain/TCodeESP32#17]
Thank you for adding new commands!
And what appeared: Pause toggle
But, as I have written everywhere more than once, the most important thing is missing: Start\Break toggle
The device not only needs to be stopped, but also for it to rise to the highest point and make it possible to get out of it.
Start function, smoothly lowering the device onto the device, to the lowest point. So that you can enter it and know the final boundary.
I downloaded the sources, I can compile, but I don’t understand the code, unfortunately...
I hope that you will listen and implement this very necessary functionality...
Well, a pause command so that commands can be executed sequentially
We press the key, the first command is executed, then after the specified time the second and so on.
Sincerely!

**Comment by jcfain** [comment]
Yes Ive thought about doing something like a pause command. Its just the code isnt setup like that and will need allot of changes to make it do so.
Interesting about a Start/Break commands. I may add those in there as they would be relatively easy. As far as the final boundry well this will be different per the application in use and no was for me to know it. Maybe if I can get firmware boundries someday.
For now, If you want to raise the device after pause, you could just add a standard tcode command L09999S1000 or what ever max/speed you want after the pause command.
